config/image-relocation-webhook/400-crds.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: imagemaps.webhook.image-relocation.pivotal.io
```

`webhook` is an implementation detail. `-` should generally be avoided in api groups.

```suggestion
  name: imagemaps.imagerelocation.pivotal.io
```

---

Removed, but kept the `-` for readability.